enterprise_access/apps/bffs/checkout/context.py

```diff
@@ -12,17 +14,47 @@ class CheckoutContext(BaseHandlerContext):
     enterprise customer data, and field constraints.
     """
 
-    def __init__(self, request):
-        """
-        Initialize the checkout context with a request.
+    @property
+    def existing_customers_for_authenticated_user(self):
+        return self.data.get('existing_customers_for_authenticated_user', [])
 
-        Args:
-            request: The HTTP request
-        """
-        super().__init__(request)
-        self.existing_customers_for_authenticated_user = []
-        self.pricing = {}
-        self.field_constraints = {}
```

Comment:
I realized yesterday at some point that I was "abusing" the contract for BFF context objects by plopping data vars right onto the Context object. This change with the getter and setter properties rectifies that, while remaining backwards-compatible and providing easy access to the data objects we care about.